### DISCUSSION TITLE
Microsoft SQL Server connect  with  Azure

### DISCUSSION THREAD

**Discussion by MohammedALREAI** [prisma/prisma#9236] [Q&A]
I have a database  in Azure so I want  to  connect  with  my app

```prisma
datasource db {
  provider = "sqlserver"
  url      =
 "sqlserver://MY-HOST;initial catalog=prisma3;user=mohammedalreai;password=1512345aaAA@;trustServerCertificate=true"

}

generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["interactiveTransactions"]
}

model Boat {
  id                Int                @id @default(autoincrement())
  dealershipId      Int?

}
```

i see this  error  
```
Datasource "db" - SQL Server

Error: P3020

The automatic creation of shadow databases is disabled on Azure SQL. Please set up a shadow database using the `shadowDatabaseUrl` datasource attribute.
Read the docs page for more details: https://pris.ly/d/migrate-shadow
```

**Comment by ryands17** [comment]
@MohammedALREAI 👋 

From the error, it seems that you need to setup a shadow database for migrations to work as shown [here](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually).

**Reply by janpio** [reply]
Alternatively you could use `npx prisma db push` instead of `prisma migrate` commands to create the schema in your remote database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
